### DISCUSSION TITLE
EGUI ready to push to crates.io?

### DISCUSSION THREAD

**Discussion by John-Nagle** [emilk/egui#4989] [General]
Github master is at 22.1.0 but crates.io is at egui = "0.28.1".

Could we get a push to crates.io, please? Is 22.1.0 stable enough yet?

(Use case: want to push a crate of mine to crates.io, and it uses wgpu and egui.  You can't push to crates.io with a dependency outside of crates.io.)

**Comment by bn-zr** [comment]
The master is also on `0.28.1` https://github.com/emilk/egui/blob/9a1e358a144b5d2af9d03a80257c34883f57cf0b/Cargo.toml#L26

**Reply by John-Nagle** [reply]
There are currently **two incompatible versions of egui numbered 0.28.1.** 

- [The one on Github imports wgpu 22.0.0.](https://github.com/emilk/egui/commit/56df31ab489312014d57247db438ecd7189a710a)
- The one on crates.io imports an older, incompatible version of wgpu. 

See discussion at [Issue 4850](https://github.com/emilk/egui/issues/4850) The [pull request which fixed that](https://github.com/emilk/egui/pull/4847) did not change the egui version.
